content/docs/for-buildpack-authors/concepts/build-plan.md

```diff
@@ -3,8 +3,42 @@ title="What is the build plan?"
 weight=99
 +++
 
+A **build plan** is a `toml` file is the output of the [detect](https://buildpacks.io/docs/for-buildpack-authors/concepts/lifecycle-phases/#phase-2-detect) phase, in which each image extension or component buildpack may express the dependencies it requires and the dependencies it provides.
+
 <!--more-->
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+## Example Build Plan (TOML)
+
+In order to make contributions to the Build Plan, a `/bin/detect` executable MUST write entries to `<plan>` in two sections: `requires` and `provides`.
+Additionally, these two sections MAY be repeated together inside of an `or` array at the top-level. Each `requires` and `provides` section MUST be a list of entries formatted as described below:
```

Comment:
Do we need to be clear here where the `<plan>` resides?  As a buildpack author I'd expect this to be within the layers that I am creating.

Reply:
Agree; updated